src/main/java/com/epam/reportportal/cucumber/AbstractReporter.java

```diff
@@ -278,34 +294,54 @@ private void removeFromTree(RunningContext.FeatureContext featureContext, Runnin
 	 */
 	@Nonnull
 	@SuppressWarnings("unused")
-	protected FinishTestItemRQ buildFinishTestItemRequest(@Nonnull Maybe<String> itemId, @Nullable ItemStatus status) {
+	protected FinishTestItemRQ buildFinishTestItemRequest(@Nonnull Maybe<String> itemId, @Nullable ItemStatus status,
+														  @Nullable Throwable error) {
 		FinishTestItemRQ rq = new FinishTestItemRQ();
+		if (status == ItemStatus.FAILED) {
+			Optional<String> currentDescription = Optional.ofNullable(descriptionsMap.get(itemId.blockingGet()));
```

Comment:
I do not allow to use `blockingGet` here, please rewrite.

Reply:
The issue was fixed in commit b4c87fd1b8efa718a2e6076370b40a3b1e3274c8
Thank you !